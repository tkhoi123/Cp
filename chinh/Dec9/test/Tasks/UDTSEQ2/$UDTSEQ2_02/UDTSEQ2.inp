2 5
1 1 11981
1 1 -18717
2 2 21396
1 1 27201
2 2 11928
