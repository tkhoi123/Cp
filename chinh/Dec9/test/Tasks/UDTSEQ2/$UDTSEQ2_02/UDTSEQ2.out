20465 33324 